index.html

```diff
@@ -0,0 +1,208 @@
+          <div class="description">
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+          </div>
+          <div class="button-links">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+          </div>
+          <div class="project">See Project</div>
+        </div>
+      </div>
+
+      <div class="card">
+        <img class="img" src="image/card-2.svg" alt="">
+        <div class="content">
+          <div class="topic">Multi-Post <br /> Stories</div>
+          <div class="sub-topic">
+            <span>CANOPY</span>
+            <span class="dot"></span>
+
+            <span>Back End Dev</span>
+            <span class="dot"></span>
+
+            <span>2015</span>
+          </div>
+          <div class="description">
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+          </div>
+          <div class="button-links">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+          </div>
+          <div class="project">See Project</div>
+        </div>
+      </div>
+
+      <div class="card">
+        <img class="img" src="image/card-3.svg" alt="">
+        <div class="content">
+          <div class="topic">Tonic</div>
+          <div class="sub-topic">
+            <span>CANOPY</span>
+            <span class="dot"></span>
+
+            <span>Back End Dev</span>
+            <span class="dot"></span>
+
+            <span>2015</span>
+          </div>
+          <div class="description">
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+          </div>
+          <div class="button-links">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+          </div>
+          <div class="project">See Project</div>
+        </div>
+      </div>
+
+      <div class="card">
+        <img class="img" src="image/card-4.svg" alt="">
+        <div class="content">
+          <div class="topic">Multi-Post <br /> Stories</div>
+          <div class="sub-topic">
+            <span>CANOPY</span>
+            <span class="dot"></span>
+
+            <span>Back End Dev</span>
+            <span class="dot"></span>
+
+            <span>2015</span>
+          </div>
+          <div class="description">
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+          </div>
+          <div class="button-links">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+          </div>
+          <div class="project">See Project</div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div>
```

Comment:
- [ ] I think you should consider using a `<section> `tag for the container for the cards instead of a `<div>` tag because the `<section>` tag adds semantic meaning to its content which is the thematic grouping of elements. It, therefore, treats the section as a standalone group which is good for browser accessibility 💻 .